src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react'
-
+import React, { useEffect, useState } from 'react'
 import { useParams } from 'react-router'
 
+import { getOrder, RawOrder } from 'api/operator'
+
 export const OrderWidget: React.FC = () => {
+  const [order, setOrder] = useState<RawOrder | null>(null)
+  const [error, setError] = useState('')
+
   const { orderId } = useParams<{ orderId: string }>()
 
-  return <div>You are viewing order id: {orderId}</div>
+  useEffect(() => {
+    // TODO: get network from a hook or something
+    getOrder({ networkId: 4, orderId })
+      .then((order) => {
+        console.log('got order', order)
+        setOrder(order)
+        setError('')
+      })
+      .catch((e) => setError(e.message))
+  }, [orderId])
+
+  return (
+    <div>
+      <h1>Details for order id: {orderId}</h1>
+      {order && <p>{JSON.stringify(order)}</p>}
+      {error && <p>{error}</p>}
```

Comment:
I would consider making the error closeable

Reply:
def not for this PR just thinking out loud

Reply:
so maybe add a `// TODO ...` ? :thinking:

Reply:
Yeah, it's definitely just for testing. Did not spend time on this component, right now it's just a place holder to display the details. Its time will come ;)

Reply:
Yes, I think the design is not closed, so probably not too worth spending time on laying out. Although probably will be as david says...